drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/main/java/org/drools/workbench/screens/scenariosimulation/backend/server/ScenarioRunnerServiceImpl.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.drools.workbench.screens.scenariosimulation.backend.server;
+
+import java.util.ArrayList;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.event.Event;
+import javax.inject.Inject;
+
+import org.drools.workbench.screens.scenariosimulation.model.ScenarioSimulationModel;
+import org.drools.workbench.screens.scenariosimulation.service.ScenarioRunnerService;
+import org.guvnor.common.services.shared.test.TestResultMessage;
+import org.jboss.errai.bus.server.annotations.Service;
+import org.uberfire.backend.vfs.Path;
+
+@Service
+@ApplicationScoped
+public class ScenarioRunnerServiceImpl
+        implements ScenarioRunnerService {
+
+    @Inject
+    private Event<TestResultMessage> defaultTestResultMessageEvent;
+
+    public ScenarioRunnerServiceImpl() {
+    }
+
+    @Override
+    public void runAllTests(final String identifier,
+                            final Path path) {
+
+        defaultTestResultMessageEvent.fire(
+                new TestResultMessage(
+                        identifier,
+                        1,
```

Comment:
Can we use two variables instead of just values (here and in the following methods)?

Reply:
Current test result is mocked because the runner has not been implemented yet. As soon as the runner will be implemented, all these fake numbers will be replaced by the real ones.
That's also why you see `1 test(s) ran in 0 minutes 0 seconds.`: if you change here the number to 0 or to 10 you can show whatever you want

Reply:
Yes. This is the reason they are as they are. Giving some result to the UI. We should try to keep the number of magic numbers on minimum and use static variables for numbers like this, but this to me is the cleanest mock we can do without investing too much on a mock.

Reply:
@Rikkola, @danielezonca, alright, thanks for explaining!